resources/views/forms/themes/inline/markdown.twig

```diff
@@ -0,0 +1,30 @@
+<div role="tabpanel" class="tab-pane active" id="markdown">
+  <div class="editor-4play" id="editor">
+  </div>
+
+  name:{{ name }}
+  value:{{ value }}
+</div>
+
+{% block body %}
+  <script src="{{ cdn('js/legacy/riddled_4play.4cb8ede83122925b7e42.js') }}"></script>
```

Comment:
Ta linia nie jest tu potrzebna. 

Ta strona dolacza juz plik `legacy.js` ktory to z kolei zawiera w sobie ten plik.

Reply:
Nie wiem o co chodzi, ale jak wprowadzam zmiany w `legacy.js` to w ogóle nie widzę żeby webpack je analizował - ciągle widze stary kod.

Reply:
hmm, u mnie normalnie dziala. odpalam `yarn watch` i koduje. mozesz zaobserwowac w konsoli ze zmiana w tym pliku powoduje ze webpack ponownie kompiluje ten plik. u mnie ma 500 kB. 

Niemniej jedna przegladarka rzuca wyjatek:

```
riddled_4play.4cb8ede83122925b7e42.js:1 Uncaught TypeError: Cannot set properties of undefined (setting 'riddled_4play')
    at riddled_4play.4cb8ede83122925b7e42.js:1:187
    at Module.<anonymous> (riddled_4play.4cb8ede83122925b7e42.js:1:2)
    at Module.gz3g (legacy-e0d62341a2e01544e311.js:201:30)
    at __webpack_require__ (bootstrap:89:1)
    at Module.EXPh (legacy.js:1:1)
    at __webpack_require__ (bootstrap:89:1)
    at checkDeferredModules (bootstrap:45:1)
    at Array.webpackJsonpCallback [
```

Reply:
@adam-boduch No właśnie, `riddled_4play`, to samo jest u mnie, tzn libka jednak nie została wczytana.